lvmd/lvservice.go

```diff
@@ -306,14 +306,39 @@ func (s *lvService) ResizeLV(_ context.Context, req *proto.ResizeLVRequest) (*pr
 		return nil, status.Error(codes.OutOfRange, "shrinking volume size is not allowed")
 	}
 
-	free, err := vg.Free()
-	if err != nil {
-		log.Error("failed to free VG", map[string]interface{}{
-			log.FnError: err,
-			"name":      req.GetName(),
-		})
-		return nil, status.Error(codes.Internal, err.Error())
+	free := uint64(0)
+	var pool *command.ThinPool
+	switch dc.Type {
+	case TypeThick:
+		free, err = vg.Free()
+		if err != nil {
+			log.Error("failed to get free bytes", map[string]interface{}{
+				log.FnError: err,
```

Comment:
Is `"name": req.GetName()` intentionally dropped?

Reply:
I copied the code block from https://github.com/topolvm/topolvm/blob/5df9e7cede433d41c9c77827b0f26dacfa8a2a59/lvmd/lvservice.go#L52

I can add the "name" field if required.

Reply:
Understood. Thanks.